live-build/misc/live-build-hooks/82-upgrade-repository.binary

```diff
@@ -149,6 +149,22 @@ aptly repo add upgrade-repository ./*.deb
 
 aptly publish repo -skip-signing upgrade-repository
 
-tar -czf "$APPLIANCE_VARIANT.upgrade.tar.gz" -C upgrade-scripts . -C ~/.aptly .
+#
+# We also want to include a file about the current image's version metadata:
+#   DLPX_VERSION: The application (product) version.
+#   DLPX_MIN_VERSION: The minimum version an engine must be on, in order to
+#     upgrade to this image.
+#   DLPX_MIN_REBOOTED_VERSION: The minimum version an engine did a reboot
+#     upgrade to, in order to upgrade to this image.
+#
+rm -rf version.info
+cat <<-EOF >version.info
```

Comment:
What do you think about creating a `template.info` file in `live-build/misc/live-build-hooks`, similar to `template.ovf` and `template.ctl` to store this file? Then, we can copy it and use `sed` to replace the `DLPX_VERSION` value, similar to what we do for `template.ctl` earlier in this script.

I ask because I wonder if that'd make it more clear/easier to maintain the versions that are infrequently changing. Having a separate "template" file might make it easier to point folks to it, to show them the current values, as well as make it easier for somebody not familiar with this repository to change them (e.g. a release lead).

What do you think?